config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     inFile                     : 'data.js'
   , outFile                    : 'data-min.js'
     // for now; need to tweak this later
-  , githubLastPushCutoffMonths : 48
+  , githubLastPushCutoffMonths : 60
```

Comment:
I considered removing the time cutoff entirely but settled for increasing the limit by a bit longer than the time period since the last make (12 months extension vs 6 months since last make)

Reply:
I see it the same way
I have already used higher values for the last builds